README.MARCJOSEPH.md

```diff
@@ -0,0 +1,32 @@
+# Car Rules 
+general Usage Rules
+
+1.	The vehicle can only go to church, school, and restaurants. All other destinations require permission.
+2. Passengers are limited to Mom and/or Dad. No other passengers (including siblings) are currently allowed. Exceptions require permission for each occurrence.
+3.	The car must be kept clean, no trash or food will be present upon exiting the vehicle.
+
+
+## Gas Maintenance and Fueling
+maintenance considerations
+
+1. Gas will be provided only for trips to and from school,church, work or job interviews; any personal outings are your financial responsibility. For personal outings not related to schooling or the above, fuel usage must be calculated in advance and paid for by you. Parents will contribute $100 for maintenance every two months and $60 for fuel weekly, the car may not be driven unless it meets maintenance standards. The car should never be brought home with less than 1/4 of a tank. Insurance for the car is your responsibility.
+### Scheduling 	
+blurb
+
+1.	Your schedule must be edited and tuned for the day before using the car.
```

Comment:
and you must leave to your destination at least 10 minutes prior to the time "maps" says it will take to get there (ie: maps says that it will take 30 minutes to get there, you must leave 40 minutes prior to your appointment)